Userland/DevTools/UserspaceEmulator/SoftCPU.cpp

```diff
@@ -1665,7 +1682,12 @@ void SoftCPU::FIADD_RM32(const X86::Instruction& insn)
     fpu_set(0, fpu_get(0) + (long double)m32int);
 }
 
-void SoftCPU::FCMOVB(const X86::Instruction&) { TODO_INSN(); }
+void SoftCPU::FCMOVB(const X86::Instruction& insn)
+{
+    VERIFY(insn.modrm().is_register());
+    if (cf())
+        fpu_set(0, fpu_get(insn.rm() & 7)); // why is this the way to get "the second opperand"?
```

Comment:
Please spell check your comments.
opperand -> operand

Reply:
ups,

fixed that...